README.md

```diff
@@ -3,4 +3,31 @@ Plugin Installation Manager Tool
 
 [![Join the chat at https://gitter.im/jenkinsci/plugin-installation-manager-cli-tool](https://badges.gitter.im/jenkinsci/plugin-installation-manager-cli-tool.svg)](https://gitter.im/jenkinsci/plugin-installation-manager-cli-tool?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-The plugin manager will download plugins and their dependencies into a folder so that they can easily be imported into an instance of Jenkins. This tool with replace the install-plugins.sh script. 
+The plugin manager will download plugins and their dependencies into a folder so that they can easily be imported into an instance of Jenkins. This tool will replace the [Docker install-plugins.sh script](https://github.com/jenkinsci/docker/blob/master/install-plugins.sh).
+
+### Usage
+
+#### Getting Started
+```
+mvn clean install -Dfindbugs.skip=true
+java -jar plugin-management-cli/target/plugin-management-cli-1.0-SNAPSHOT-jar-with-dependencies.jar /file/path/jenkins.war -pluginTxtPath /file/path/plugins.txt -plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
+```
+
+#### CLI Options
```

Comment:
Note if any of the CLI options are optional.

Reply:
Fixed in 1344073.